tests/async.rs

```diff
@@ -93,24 +93,35 @@ where
 
     fn test<C, F>(self, c: C)
     where
-        F: IntoFuture<Item = (), Error = mysql_async::errors::Error>,
-        C: FnOnce(mysql_async::Conn) -> F,
+        F: IntoFuture<Item = (), Error = mysql_async::error::Error> + 'static,
+        <F as futures::IntoFuture>::Future: std::marker::Send,
+        C: FnOnce(mysql_async::Conn) -> F + Send + 'static,
     {
+        let mut runtime = Runtime::new().unwrap();
+
         let listener = net::TcpListener::bind("127.0.0.1:0").unwrap();
         let port = listener.local_addr().unwrap().port();
+
         let jh = thread::spawn(move || {
             let (s, _) = listener.accept().unwrap();
+
```

Comment:
Any particular reason for these extra empty lines? Are they added by rustfmt or something?

Reply:
This is a bad formatting habit of my own that I forget to clean up after. I also haven't gotten in the habit of using rustfmt to clean up before committing, which probably would have cleaned this up.